django_declarative_apis/machinery/filtering.py

```diff
@@ -67,6 +68,36 @@ def _get_unexpanded_field_value(inst, field_name, field_type):
         return {display_key: getattr(obj, display_key)}
 
 
+def _is_relation(field):
+    return field.is_relation or getattr(field, "is_fake_relation", False)
+
+
+def _is_reverse_relation(field):
+    return isinstance(field, ForeignObjectRel)
+
+
+def _cache_related_instance(inst, field_name, model_cache):
+    try:
+        field_meta = inst._meta.get_field(field_name)
+    except (AttributeError, FieldDoesNotExist):
+        return
```

Comment:
I'm curious why we're returning None here, shouldn't we bubble up the error? The user is trying to cache a field that doesn't exist

Reply:
The problem is that `field_name` is a consumer defined string for specifying the output field name after filtering it. So it can have aj different name of the actual field. For example:

```python
FILTERS = {
    'my_increased_counter': lambda inst: inst.counter.count + 1
}
```

Here `field_name` would be `my_increased_counter` which doesn't exists in the model.

Reply:
Right, but shouldn't we alert the user that the field doesn't exist somehow?

Reply:
Code won't break if the field doesn't exists. It just won't get cached.

Or do mean alerting the user that using `my_increased_counter` as field name he won't get caching? In that case I'm not sure how much useful it is but yeah we could add some logging

Reply:
I think I understand what you're saying now. This PR looks good, approved!